client/src/locale/en/pages.js

```diff
@@ -63,8 +63,12 @@ export default {
     'center-on': "Center on",
     'this-event-is-past': "This event is past.",
     'this-event-is-currently-running': "This event is currently running.",
+    'this-event-is-future': "This event is scheduled in the future.",
     'this-event-is-confirmed': "This event is confirmed.",
-    'this-event-is-locked-past-confirmed': "This event is locked because it's confirmed, and past.",
+    'this-event-isnot-confirmed': "This event is not confirmed yet!",
+    'this-event-is-closed': "This event is closed.",
+    'this-event-isnot-closed': "This event is not closed yet! Please consider closing it.",
```

Comment:
`this-event-is-not-closed`

Aussi, je ne sais pas si le `Please consider closing it.` est vraiment utile ...

Reply:
`this-event-is-not-confirmed` également